_haikus/unimaginative_cloud.md

```diff
@@ -0,0 +1,8 @@
+--- layout: haiku
```

Comment:
Jekyll format here is looking good, just make sure there is a return before `layout`:
```
---
layout: haiku
```